models/model.py

```diff
@@ -54,7 +60,16 @@ def __init__(self, model_hparams, optimizer_hparams):
         self.lr_scheduler = None
 
     def configure_optimizers(self):
-        optimizer = AdamW(self.parameters(), lr=self.hparams.optimizer_hparams['lr'],
+        params = list(self.named_parameters())
+
+        def is_encoder(n): return n.startswith('model')
+
+        grouped_parameters = [
+            {'params': [p for n, p in params if is_encoder(n)], 'lr': self.hparams.optimizer_hparams['lr']},
+            {'params': [p for n, p in params if not is_encoder(n)], 'lr': self.hparams.optimizer_hparams['lr'] * 100}
```

Comment:
Shouldn't we be able to specify the different learning rates ourselves? Why is it times 100?

Reply:
Ideally yes, but because we already started training without having 2 different parameters for the 2 learning rates, it would mean we have to redo the hyperparameter search with the 2 learning rate. It also reduce length of the hyperparameter search (for roberta each combination of parameters takes between 1-3h).
times 100 beause the classifier layers are not trained so the update steps should be larger. Why 100 specifically, it's arbitrary. I've seen some implementation using that so I went with that.

Reply:
But you started training just on the train task right? We didn't need an extra learning rate there. I agree that it complicates the hyperparameter search so maybe we could just go with one of which we think it will work otherwise we won't have any results tomorrow for Roberta. (Training on GNN takes only 2 minutes). If you saw this somewhere, I guess this should be fine and I would opt for using that as well. Not sure what @RedHilarious and @Matyasch think about this?

Reply:
So I did not think that the * 100 is arbitrary. In the first trainings I did, I unfortunately froze all roberta and only trained the classifier. Here a lr of `5e-3/4` did work well. Now with Roberta we found that a lr of `5e-6`works well, so multiplying it with 100, we end up at the ideal LR for the classifier. That is my reasoning behind it.

We started training on Task 1 and yes, we also used 2 learning rates here (and I also tuned both), the original roberta one and then *100 for the classifier. 

So I think we know which LR works well for the classifier, so the only thing we still need to find out is which LR works good for the GNN.